libs/lib-client/core/src/services/restapi/v2/contributions.service.ts

```diff
@@ -24,9 +24,12 @@ export class ContributionsService {
   }
 
   listContribs<T extends ContributionSchema = ContributionSchema>(
-    type: FLOGO_CONTRIB_TYPE
+    type?: FLOGO_CONTRIB_TYPE
   ) {
-    return this.restApi.get<T[]>(this.getApiPath() + '?filter[type]=' + type).toPromise();
+    const apiPath = type
```

Comment:
Prefer using angular's `HttpParams` instead of building the query string manually: https://angular.io/guide/http#url-parameters